### PR TITLE
Adding the my notes component

### PR DESCRIPTION
**Pull request by areccus**
## Description

___Add your PR description here, deleting this example___

#### Video Link

https://www.loom.com/share/7ed04b4e9f9741498131e510547e5757

#### Contributors
Anthony "@acoman-cloud" Coman
Robert @itsmerobert12" Gomez

## Type of change

___Please delete options that are not relevant.___

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by acoman-cloud — APPROVED** [review]
Great work! Nice to see some clean up and functionality being added.

File: src/components/common/NotesTable/NoteUtils.js [hunk 1]
```diff
@@ -1,44 +1,18 @@
-import styled from 'styled-components';
-import { Button, Input, Space } from 'antd';
+import { Badge, Button, Input, Space } from 'antd';
 import { SearchOutlined } from '@ant-design/icons';
+import * as MdIcons from 'react-icons/md';
+import '../styles/Notes.css';
 
-// Flag Styling
```
Comment (morgankj) [inline src/components/common/NotesTable/NoteUtils.js:5]: I like this change of adding the styling into its own doc - much cleaner and easier to follow in my opinion

File: src/components/common/NotesTable/NoteUtils.js [hunk 2]
```diff
@@ -1,44 +1,18 @@
...
-  medium: '#FFD93D',
-  low: '#6BCB77',
-};
-
-export const statusColor = {
-  'in progress': '#2D88D9',
-  draft: '#E8833B',
-  replied: '#9964C4',
-  resolved: '#48C73A',
+// Flag Icons
```
Comment (morgankj) [inline src/components/common/NotesTable/NoteUtils.js:6]: Good comments here to clue in other devs

**Review by morgankj — APPROVED** [review]
Touched quite a few files here, which always makes me nervous XD But it looks like your changes were clean and dry! Good work following best practices too; you did a good job of removing unused imports and blocks

File: src/components/common/NotesTable/index.js [hunk 3]
```diff
@@ -9,27 +9,49 @@ import {
...
+      >
+        Save edit
+      </Button>
+      <Button onClick={onCancel}>Cancel</Button>
+    </Form.Item>
+  </>
+);
 
 const NotesTable = ({ userProfile }) => {
+  console.log(userProfile);
```
Comment (maidiance) [inline src/components/common/NotesTable/index.js:40]: This console log should be removed.

File: src/components/common/NotesTable/index.js [hunk 4]
```diff
@@ -9,27 +9,49 @@ import {
...
   // edit users own comment states
   const [editing, setEditing] = useState(false);
   const [editNote, setEditNote] = useState({ key: '', note: '' });
   const [submitting, setSubmitting] = useState(false);
   // Get profile_id of logged in user
   const { profile_id } = userProfile;
-  const { TextArea } = Input;
-
+  const location = useLocation();
+  console.log(location.pathname);
```
Comment (maidiance) [inline src/components/common/NotesTable/index.js:50]: Do not leave console logs in your code.

**Review by maidiance — CHANGES_REQUESTED** [review]
Overall looks great. Please be sure to clean up excess console logs though!

**Review by phillipgalbraith — COMMENTED** [review]
This pull request is description is effective in referencing specific reasons for changing or adding code and the methods used.  Consider moving the Notes.js inline styles to Notes.css with classes.

**Review by arsalan2243 — APPROVED** [review]
this looks clean and adding better functionality. if you remove the console logs it would be good to go.

**Review by beefybroccoli — APPROVED** [review]
Thes scope is sufficiently small. The new functionality is needed.

**Pull request merged**